### PR TITLE
Fix a bug polygon fill layer updates when line color changes

### PR DESCRIPTION
**Pull request by heshan0131**
Polygon layer fill should not update when line color update triggers change

Screen Shot 2020-10-29 at 10 22 24 PM

Signed-off-by: Shan He <heshan0131@gmail.com>

### CODE REVIEW

**Comment by Pessimistress** [comment]
Line color applies to the SolidPolygonLayer when `wireframe: true`.

**Comment by heshan0131** [comment]
> Line color applies to the SolidPolygonLayer when `wireframe: true`.

When I use `PolygonLayer` and provide getLineColor, and its updateTriggers, polygon layer renders PathLayer, passing getLineColor -> getColor. Change of lineColor triggers should only updates the PathLayer, it should not cause the solid polygon layer to update...

**Comment by Pessimistress** [comment]
That is not caused by updateTriggers, but the accessor itself. `getLineColor` is only used when `extruded: true, wireframe: true`, and it shouldn't be passed to `SolidPolygonLayer` otherwise. This bug affects both `GeoJsonLayer` and `PolygonLayer`.

Can you open an issue first?

**Comment by heshan0131** [comment]
https://github.com/visgl/deck.gl/issues/5085

File: modules/layers/src/polygon-layer/polygon-layer.js [hunk 1]
```diff
@@ -193,6 +193,9 @@ export default class PolygonLayer extends CompositeLayer {
             getPolygon: updateTriggers.getPolygon,
             getElevation: updateTriggers.getElevation,
             getFillColor: updateTriggers.getFillColor,
+            // using a legacy API to invalid lineColor attributes
+            // if (extruded && wireframe) has changed
+            lineColor: extruded && wireframe,
```
Comment (Pessimistress) [inline modules/layers/src/polygon-layer/polygon-layer.js:198]: `Boolean(extruded && wireframe)` just to be safe

**Review by Pessimistress — APPROVED** [review]

**Pull request merged**